src/devices/bus/coco/coco_t4426.h

```diff
@@ -0,0 +1,56 @@
+// license:BSD-3-Clause
+// copyright-holders:Nathan Woods
+#pragma once
+
+#ifndef __COCO_T4426_H__
```

Comment:
Please don't add more __NAME macros to the codebase.  All identifiers with __ in them are reserved.  Make the macro MAME_BUS_COCO_T4426_H or something.  Also, please move the #pragma once inside the conventional #include guards to help compilers that don't recognise this pragma.